pom.xml

```diff
@@ -92,13 +92,6 @@
         <groupId>com.infradna.tool</groupId>
         <artifactId>bridge-method-injector</artifactId>
         <version>1.14</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>process</goal>
-            </goals>
-          </execution>
-        </executions>
```

Comment:
Nop. The execution is needed. Otherwise the goal is never invoked in the lifecycle.